vembrane/ann_types.py

```diff
@@ -43,11 +45,16 @@ def __getitem__(self, spec):
 IntFloatStr = Union[int, float, str]
 
 
-def type_info(value):
+def type_info(value, number="."):
     if value is None:
         return NA
     if isinstance(value, tuple):
-        return InfoTuple(value)
+        if number != "A":
+            return InfoTuple(value)
+        else:
+            if len(value) != 1:
+                raise MoreThanOneAltAllele()
+            return value[0]
```

Comment:
This branch doesn't do the `None` to `NA` substitution.

Reply:
Good catch!